docs/src/examples/components/Attachment/Variations/AttachmentActionableExample.shorthand.tsx

```diff
@@ -2,18 +2,21 @@ import * as React from 'react'
 import { Attachment } from '@stardust-ui/react'
 
 class AttachmentActionableExampleShorthand extends React.Component {
-  handleClick = () => alert('Attachment was clicked')
+  handleClick = message => e => {
+    alert(`'${message}' was clicked`)
+    e.stopPropagation()
+  }
 
   render() {
     return (
       <Attachment
         actionable
-        icon="file word outline"
+        icon="table"
         header="Document.docx"
         description="800 Kb"
-        action={{ icon: 'x' }}
+        action={{ icon: 'more', onClick: this.handleClick('More Action') }}
         progress={33}
-        onClick={this.handleClick}
+        onClick={this.handleClick('Attachment')}
```

Comment:
We should not use `alert()` in docs, can we implement logging like it is done there?

https://stardust-ui.github.io/react/components/portal#types-portal
https://github.com/stardust-ui/react/edit/master/docs/src/examples/components/Portal/Types/PortalExample.shorthand.tsx?message=docs(PortalExample.shorthand):%20your%20description

Reply:
@layershifter we do have examples that use `alert()`; maybe we should add a chore task to replace all with logging